index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav class="container">
+
+    <div class="time-cell-wifi-batt">
+      <div class="time">9:41</div>
+      <div class="cell"><img src="image/cell.png" alt="cell"></div>
+      <div class="wifi"><img src="image/wifi.png" alt="wifi"></div>
+      <div class="batt"><img src="image/batt.png" alt="batt"></div>
+    </div>
+    <div class="menu"><img src="image/menu-btn.png"></div>
+    <div class="logo"><img src="image/my-logo.png" alt="My Logo"></div>
+
+    <div class="container-headline">
+      <div class="primary-text">
+        Hey There. <br>
+        I'm Joshua.
+      </div>
+      <div class="supporting-text">I'm a Software Developer</div>
```

Comment:
- [ ] This subheading should also make use of one of the other `<heading>` tags such as `<h2>`, `<h3>` etc :+1: